popde/density_estimate.py

```diff
@@ -0,0 +1,92 @@
+# First script for iterative KDE
```

Comment:
remove please, comments should not concern what was 'first' or not, or what we might use code for, they should describe what is _in_ the code

Reply:
removed